.circleci/config.yml

```diff
@@ -4,6 +4,7 @@ version: 2.1
 
 orbs:
   windows: circleci/windows@2.2.0
+  general-platform-helpers: cirro-eng-services/general-platform-helpers@1.2
```

Comment:
Replace `cirro-eng-services` with your own org (in this case, `okta`).

Sorry, that was not covered in the quickstart.